File: scratch/repair.js
const fs = require('fs');
const filePath = 'c:\\Users\\User\\OneDrive\\Desktop\\Path-Finder\\src\\app\\setup\\grades\\page.js';
const buf = fs.readFileSync(filePath);
const text = buf.toString('utf8');
fs.writeFileSync(filePath, text, 'utf8');
console.log('Repair completed!');

File: scratch/repair.py
with open(r"c:\Users\User\OneDrive\Desktop\Path-Finder\src\app\setup\grades\page.js", "rb") as f:
    data = f.read()

# Decode ignoring invalid characters
text = data.decode("utf-8", errors="ignore")

with open(r"c:\Users\User\OneDrive\Desktop\Path-Finder\src\app\setup\grades\page.js", "w", encoding="utf-8") as f:
    f.write(text)

print("Repair completed!")
